src/api.js

```diff
@@ -60,7 +60,6 @@ builder.declare({
   name: 'connect',
   description: 'Connect to receive messages',
   stability: APIBuilder.stability.experimental,
-  // Add input validation yml
```

Comment:
Why not do input validation?

Reply:
Input validatiion is already done here - https://github.com/taskcluster/taskcluster-events/blob/master/src/api.js#L28